src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -161,177 +132,127 @@ class WorkspaceInvitePage extends React.Component {
+        if (!validate()) {
             return;
         }
 
         const invitedEmailsToAccountIDs = {};
-        _.each(this.state.selectedOptions, (option) => {
+        _.each(selectedOptions, (option) => {
             const login = option.login || '';
             const accountID = lodashGet(option, 'participantsList[0].accountID');
             if (!login.toLowerCase().trim() || !accountID) {
                 return;
             }
             invitedEmailsToAccountIDs[login] = Number(accountID);
         });
-        Policy.setWorkspaceInviteMembersDraft(this.props.route.params.policyID, invitedEmailsToAccountIDs);
-        Navigation.navigate(ROUTES.getWorkspaceInviteMessageRoute(this.props.route.params.policyID));
-    }
-
-    /**
-     * @returns {Boolean}
-     */
-    validate() {
-        const errors = {};
-        if (this.state.selectedOptions.length <= 0) {
-            errors.noUserSelected = true;
-        }
-
-        Policy.setWorkspaceErrors(this.props.route.params.policyID, errors);
-        return _.size(errors) <= 0;
-    }
-
-    render() {
-        const headerMessage = OptionsListUtils.getHeaderMessage(this.state.personalDetails.length !== 0, Boolean(this.state.userToInvite), this.state.searchTerm);
-        const policyName = lodashGet(this.props.policy, 'name');
-
-        return (
-            <ScreenWrapper shouldEnableMaxHeight>
-                {({didScreenTransitionEnd}) => {
-                    const sections = didScreenTransitionEnd ? this.getSections() : [];
-                    return (
-                        <FullPageNotFoundView
-                            shouldShow={_.isEmpty(this.props.policy)}
-                            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
-                        >
-                            <HeaderWithBackButton
-                                title={this.props.translate('workspace.invite.invitePeople')}
-                                subtitle={policyName}
-                                shouldShowGetAssistanceButton
-                                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
-                                onBackButtonPress={() => {
-                                    this.clearErrors();
-                                    Navigation.goBack(ROUTES.getWorkspaceMembersRoute(this.props.route.params.policyID));
-                                }}
+        Policy.setWorkspaceInviteMembersDraft(props.route.params.policyID, invitedEmailsToAccountIDs);
+        Navigation.navigate(ROUTES.getWorkspaceInviteMessageRoute(props.route.params.policyID));
+    };
+
+    const headerMessage = OptionsListUtils.getHeaderMessage(personalDetails.length !== 0, Boolean(userToInvite), searchTerm);
+    const policyName = lodashGet(props.policy, 'name');
+    const shouldShowAlertPrompt = _.size(lodashGet(props.policy, 'errors', {})) > 0 || lodashGet(props.policy, 'alertMessage', '').length > 0;
```

Comment:
can we `useMemo` or keep it as method as it was before

Reply:
`useMemo` has 3 use cases. 
1. Expensive calculation
2. For props of memoized components
3. Dependency to other hooks

I don't think we need to use `useMemo` here.

Reply:
NVM, my bad I overlooked it.